### PR TITLE
Added reading from 2 sensors using Dallas library

### PR DESCRIPTION
**Pull request by ihor96berizka**

### CODE REVIEW

File: data-temp-dallas.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+void printAddress(DeviceAddress deviceAddress){
+  for (uint8_t i = 0; i < 8; i++)
+  {
+    if (deviceAddress[i] < 16) Serial.print("0");
+    Serial.print(deviceAddress[i], HEX);
+  }
+}
+
+float getTemperature(DeviceAddress deviceAddress){
```
Comment (shybzzz) [inline data-temp-dallas.h]: I would remove this method as we do not need so verbose loging to serial

File: data-temp-dallas.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#ifndef __DATA_DALLAS_TEMPERATURE__
+#define __DATA_DALLAS_TEMPERATURE__
+
+#include <DallasTemperature.h>
+
+#define SENSOR_COUNT 2u
+#define SENSOR_RESOLUTION 12u
+
+DallasTemperature dallasSensors(&ds);
```
Comment (shybzzz) [inline data-temp-dallas.h]: Can we instantiate 1 more 1wire here to avoid dependency on data-temp?

File: data-temp-dallas.h [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    dallasSensors.setResolution(sensorAddress[i], SENSOR_RESOLUTION);
+  }
+}
+
+int getDallasTempData()
+{
+  dallasSensors.requestTemperatures(); // ds18b20 calculates temrepature
+  //  get temperature
+  return getTemperature(sensorAddress[1]) - getTemperature(sensorAddress[0]);
```
Comment (shybzzz) [inline data-temp-dallas.h]: Check whether sensorAdress has 2 items
What will happen in case I plug esp into board without sensors at all?

**Review by shybzzz — COMMENTED** [review]
This can be merged after comments are fixed

File: data-temp-dallas.h (hunk 2 above)
Comment (ihor96berizka) [reply data-temp-dallas.h]: Do we need 2 instances that do the same?
As we include both headers in *.ino file - we always have this variable.

**Pull request closed without merging**